includes/data-events/connectors/class-mailchimp.php

```diff
@@ -181,13 +181,15 @@ public static function put( $email, $data = [] ) {
 			'status_if_new' => 'transactional',
 		];
 
-		$merge_fields = self::get_merge_fields( $audience_id, $data );
+		// Normalize contact metadata.
```

Comment:
Don't we need to check if the class exists here? What if Newsletters plugin is not present?

Reply:
In this case `Newspack_Newsletters` refers to the namespaced class in this plugin: https://github.com/Automattic/newspack-plugin/blob/master/includes/data-events/connectors/class-mailchimp.php#L13